client-course-schedulizer/src/components/reuseables/GridItem/GridItemCheckboxGroup/GridItemCheckboxGroup.tsx

```diff
@@ -0,0 +1,39 @@
+import { Checkbox, FormControlLabel, FormLabel, Grid } from "@material-ui/core";
+import React from "react";
+import { useForm } from "react-hook-form";
+import "./GridItemCheckboxGroup.scss";
+
+export const GridItemCheckboxGroup = ({
+  label,
+  name,
+  options,
+  register,
+}: {
+  label: string;
+  name?: string | undefined;
```

Comment:
Out of curiosity, since name is optional, doesn't it allow for undefined as a type, or does it need to be specified? This isn't an issue, I'm just wondering.

Reply:
Yep, you're right